### PR TITLE
Tlg2178.tlg002

### PR DESCRIPTION
**Pull request by planatheisa**
Checking Hook.

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `coverage`         |    -0.01 |
| `metadata_passing` |     2    |
| `metadata_total`   |     2    |
| `nodes_count`      |    24    |
| `texts_total`      |     1    |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |     +675 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg2178/__cts__.xml`                          | New      |
| `./data/tlg2178/tlg002/__cts__.xml`                   | New      |
| `./data/tlg2178/tlg002/tlg2178.tlg002.1st1K-grc1.xml` | New      |
| `./data/tlg2178/__cts__.xml`                          | Passing  |
| `./data/tlg2178/tlg002/__cts__.xml`                   | Passing  |
| `./data/tlg2178/tlg002/tlg2178.tlg002.1st1K-grc1.xml` | Failing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3754)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +24 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |     +675 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg2178/__cts__.xml`                          | New      |
| `./data/tlg2178/tlg002/__cts__.xml`                   | New      |
| `./data/tlg2178/tlg002/tlg2178.tlg002.1st1K-grc1.xml` | New      |
| `./data/tlg2178/__cts__.xml`                          | Passing  |
| `./data/tlg2178/tlg002/__cts__.xml`                   | Passing  |
| `./data/tlg2178/tlg002/tlg2178.tlg002.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3760)*

File: data/tlg2178/tlg002/tlg2178.tlg002.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title xml:lang="lat">Rhetores Graeci ex Recognitione</title>
+                            <editor>
+                                <persName>
+                                    <name xml:lang="lat">Leonardi Spengel</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:tmp"></author>
```
Comment (sonofmun) [inline data/tlg2178/tlg002/tlg2178.tlg002.1st1K-grc1.xml]: Need to add author name and URN (probably also @xml:lang attribute).

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +24 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |     +675 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg2178/__cts__.xml`                          | New      |
| `./data/tlg2178/tlg002/__cts__.xml`                   | New      |
| `./data/tlg2178/tlg002/tlg2178.tlg002.1st1K-grc1.xml` | New      |
| `./data/tlg2178/__cts__.xml`                          | Passing  |
| `./data/tlg2178/tlg002/__cts__.xml`                   | Passing  |
| `./data/tlg2178/tlg002/tlg2178.tlg002.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3803)*

**Pull request merged**